subprojects/packagefiles/cmark-gfm/cmark-gfm-extensions_export.h.meson

```diff
@@ -0,0 +1,38 @@
+#ifndef CMARK_GFM_EXTENSIONS_EXPORT_H
+#define CMARK_GFM_EXTENSIONS_EXPORT_H
+
+/* Is this an exclusively static build */
+#if @CMARK_GFM_STATIC_DEFINE@ && ! defined CMARK_GFM_STATIC_DEFINE
+#  define CMARK_GFM_STATIC_DEFINE
+#endif
+
+/*
+ * Here is the complicated part. Windows is special -- you cannot just define
+ * entry points unconditionally.
+ * */
+#if defined _WIN32 || defined __CYGWIN__
+   /* When building static libraries, avoid marking public ones */
+#  if defined CMARK_GFM_STATIC_DEFINE
+#    define CMARK_GFM_EXTENSIONS_EXPORT
+   /* We are building this library */
+#  elif defined libcmark_gfm_EXPORTS
```

Comment:
Ah right, there's some MSVC warnings about inconsistent linkage because these defines are not passed at the time of building.

Reply:
I believe the most recent commit should fix this issue by adding `-Dlibcmark_gfm_EXPORTS` as a `cpp_arg`. Let's see if it passes CI.

Reply:
Right. CMake automatically does that by default just in case you want to use generated export headers (and in our case, we are using templated export headers) and I didn't think to check that that was implemented here.